### DISCUSSION TITLE
session in load function isn't typed

### DISCUSSION THREAD

**Discussion by lkj4** [sveltejs/kit#5189] [Q&A]
I get

`[ts 7031] [E] Binding element 'session' implicitly has an 'any' type`

at 

```typescript
export async function load({ session }) { ...
```

and I have `app.d.ts`:

```typescript
/// <reference types="@sveltejs/kit" />

declare global {
  declare namespace App {
    interface Locals {
      ...
```

and I have `"strict": true` in tsconfig.

**Comment by westy92** [comment]
You can fix this by following these instructions: https://github.com/sveltejs/kit/issues/6926#issuecomment-1252939357
